virttest/utils_test/__init__.py

```diff
@@ -2626,6 +2626,50 @@ def check_dest_vm_network(vm, vm_ip, remote_host, username, password,
                                   % (vm.name, results_stdout_52lts(result)))
 
 
+def check_remote_vm_disks(vm, params):
```

Comment:
can we keep it in virttest/utils_disk.py ?

Reply:
Yes,  I will update.

Reply:
Updated